docker/docker.settings

```diff
@@ -125,9 +125,7 @@ COMPRESS_OFFLINE = False
 
 LOGGING["root"]["level"] = os.environ.get("DJANGO_LOG_LEVEL", "INFO")
 
-RAVEN_CONFIG = {
-    "dsn": os.environ.get("RAVEN_DSN")
-}
+SENTRY_DSN = os.environ.get("SENTRY_DSN", os.environ.get("RAVEN_DSN"))
```

Comment:
As we are importing the `settings_common` at the beginning of this file, I think we should have the SDK init on this file as well. Ref.: 
1. https://github.com/greatnonprofits-nfp/rapidpro/blob/24e65eb54057a67f5a2dd1b48fc3acf36818df0f/docker/docker.settings#L8
2. https://github.com/greatnonprofits-nfp/rapidpro/blob/24e65eb54057a67f5a2dd1b48fc3acf36818df0f/temba/settings_common.py#L24

Reply:
According to the words that you said about importing `settings_common`, I would rather delete this line
```
SENTRY_DSN = os.environ.get("SENTRY_DSN", os.environ.get("RAVEN_DSN"))
```
from the `docker.settings` file. I don't see any reasons to duplicate the SDK init to `docker.settings`, as it will be executed anyway. Also, I tested both variants on the local machine using `ccl-dev-env` DSN, and there was no difference. In both cases, a custom error that I produced was successfully delivered to the sentry.

Reply:
If you see it necessary I can move that part from `settings_common` to `docker.settings`

Reply:
Ok, let's keep only on settings_common, makes sense. Thanks.